include/mitsuba/render/bsdf.h

```diff
@@ -462,6 +462,10 @@ class MI_EXPORT_LIB BSDF : public Object {
     /// Set a string identifier
     void set_id(const std::string& id) override { m_id = id; };
 
+    /// Return the diffuse reflectance value (if any)
+    virtual Spectrum get_diffuse_reflectance(const SurfaceInteraction3f &si,
```

Comment:
Should we rename this method `diffuse_reflectance` or `eval_diffuse_reflectance`? Btw should this compute the diffuse reflectance or the albedo of the BSDF? IIRC the albedo can be computed as the integration of the diffuse reflectance for all incoming directions).

Reply:
:+1: Ok with the rename.

So, I'm often confused about this. My understanding is that the albedo is defined as the fraction of radiance over irradiance, but that it is usually approximated by using only the diffuse component. Also, it should be defined as a double integral over the hemisphere (both incident and outgoing direction), such that it is truly view independent.
However, this would then have to be implemented as a MC integration for most BSDFs if we wanted it to be accurate (we only have a few existing BSDFs that have a closed form solution for this double integral). So the further approximation is to only consider the normal outgoing direction. Maybe we should just pay the performance cost for the MC integration with an arbitrary number of samples that can be set in the `aov` integrator?